### PR TITLE
Improve nodejs example to show subdirectories sync

### PR DESCRIPTION
**Pull request by Ilyes-Hammadi**
I've updated the `nodejs` example to show the use of the undocumented `folder/***` sync feature that syncs subdirectories. My changes consist of just placing the javascript files in multiple folders and to get the attention of the developer learning `skaffold` to pay attention when checking the `skaffold.yaml` file on the difference between the `folder/**` and `folder/***` sync mechanisms.

### CODE REVIEW

**Comment by Ilyes-Hammadi** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2FGoogleContainerTools%2Fskaffold%2Fpull%2F2024) for more info**.

I signed it!

**Comment by Ilyes-Hammadi** [comment]
@priyawadhwa should be fixed now!

File: integration/examples/nodejs/skaffold.yaml [hunk 1]
```diff
@@ -5,7 +5,9 @@ build:
   - image: gcr.io/k8s-skaffold/node-example
     context: backend
     sync:
-      '*.js': .
+      # Sync all the javascript files that are in the src folder 
+      # with the container src folder
+      'src/***/*.js': src/
```
Comment (tejal29) [inline integration/examples/nodejs/skaffold.yaml]: Hi, @Ilyes-Hammadi , we are getting rid of "***" notation in upcoming PRs.
However, its a good example for conversion.

**Comment by tejal29** [comment]
@Ilyes-Hammadi , can you make the change in both files? so they are in both dir
examples are served as user samples and integration/examples are tested

**Comment by Ilyes-Hammadi** [comment]
@tejal29 The changes now are now in both folders.

**Pull request merged**